specification/eventgrid/resource-manager/Microsoft.EventGrid/2017-06-15-preview/EventGrid.json

```diff
@@ -0,0 +1,1760 @@
+                    "description": "A collection of EventSubscriptions",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/EventSubscription"
+                    }
+                }
+            }
+        },
+        "OperationsListResult": {
+            "description": "Result of the List Operations operation",
+            "type": "object",
+            "properties": {
+                "value": {
+                    "description": "A collection of operations",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Operation"
+                    }
+                }
+            }
+        },
+        "Operation": {
+            "description": "Represents an operation returned by the GetOperations request",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "Name of the operation",
+                    "type": "string"
+                },
+                "display": {
+                    "$ref": "#/definitions/OperationInfo",
+                    "description": "Display name of the operation"
+                },
+                "origin": {
+                    "description": "Origin of the operation",
+                    "enum": [
+                        "User",
+                        "System",
+                        "UserAndSystem"
+                    ],
+                    "type": "string",
+                    "x-ms-enum": {
+                        "name": "OperationOrigin",
+                        "modelAsString": true
+                    }
+                },
+                "properties": {
+                    "description": "Properties of the operation",
+                    "type": "object",
+                    "x-ms-client-flatten": true
+                }
+            }
+        },
+        "OperationInfo": {
+            "description": "Information about an operation",
+            "type": "object",
+            "properties": {
+                "provider": {
+                    "description": "Name of the provider",
+                    "type": "string"
+                },
+                "resource": {
+                    "description": "Name of the resource type",
+                    "type": "string"
+                },
+                "operation": {
+                    "description": "Name of the operation",
+                    "type": "string"
+                },
+                "description": {
+                    "description": "Description of the operation",
+                    "type": "string"
+                }
+            }
+        },
+        "TopicProperties": {
+            "description": "Properties of the Topic",
+            "type": "object",
+            "properties": {
+                "provisioningState": {
+                    "description": "Provisioning state of the t
```

Comment:
Sorry I didn't fully get this comment - are you referring to the provisioningState property (which is already an enum), or are you referring to TopicProperties as a whole?